### PR TITLE
Rework "Tuning Workload Memory Protection" section

### PR DESCRIPTION
**Pull request by tomschr**
This PR contains:

* Conversion from ADoc file into DocBook5
* Adapt structural changes

Related to https://jira.suse.com/browse/PM-1093

### CODE REVIEW

File: xml/s4s_tune_wmp.xml
Comment (lproven) [inline xml/s4s_tune_wmp.xml:181]:
```suggestion
                    <para>For each &sap; instance, add the following line to the
```

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/s4s_tune_wmp.xml
Comment (tomschr) [reply xml/s4s_tune_wmp.xml:168]: Good point! What do you think @scmschmidt?

Comment (scmschmidt) [reply xml/s4s_tune_wmp.xml:168]: In principle I would agree, but this comes directly from the man page of systemd.resource-control(5):

      MemoryLow=bytes
           ...
           Takes a memory size in bytes. If the value is suffixed with K, M, G or T, the specified memory size is parsed as
           Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively. Alternatively, a percentage value
           may be specified, which is taken....

Comment (tomschr) [reply xml/s4s_tune_wmp.xml:168]: Hmn, :thinking: that's a tricky question. Maybe we should leave it as it is. Deviating from the manpage could confuse people. At least the base is written (1024 instead of 1000) which should be enough, although the units are not entirely correct.

@lproven what do you think?

Comment (scmschmidt) [reply xml/s4s_tune_wmp.xml:168]: On second thought, I think we should correct it. We don't have to do the same "mistakes" as the systemd guys. ;-)
I just copied it to make my life easier.

Comment (tomschr) [reply xml/s4s_tune_wmp.xml]: Okay, you've convinced me. :wink:  I've changed that to:

```
MemoryLow takes the memory size in bytes. If
the value is suffixed with K, M, G or T, the
specified memory size is parsed as Kibibytes,
Mebibytes, Gibibytes, or Terabytes (with the base
1024 instead of 1000, see <link
  xlink:href="https://en.wikipedia.org/wiki/Binary_prefix"/>),
respectively.
```

**Comment by tomschr** [comment]
@scmschmidt  You said yesterday, we need to change the beginning of the file (there is no SAP Note anymore, some lines about how to determine `MemoryLow`). Happy to add that. :wink:

**Comment by tomschr** [comment]
Check by Sören and he's fine. We need to wait for testing to be finished before we can merge it.

**Comment by tomschr** [comment]
Final ok from Sören. :+1:

**Pull request merged**
